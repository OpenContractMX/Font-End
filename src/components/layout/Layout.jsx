import React, { Fragment } from "react";
import { Header } from "../header";
import { Footer } from "../footer";

export const Layout = (props) => {
  // const children = props.children;
  return (
    <Fragment>
      <Header />
      {props.children}
      <Footer />
    </Fragment>
  );
};
