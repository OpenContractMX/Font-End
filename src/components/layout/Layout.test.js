import React from "react";
import { BrowserRouter } from "react-router-dom";

import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { Layout } from "./Layout";

it("[render content `statistic / Layout`]", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Layout />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
