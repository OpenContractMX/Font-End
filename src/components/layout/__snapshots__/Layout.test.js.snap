// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render content \`statistic / Layout\`] 1`] = `
<DocumentFragment>
  <header
    class="header"
    id="header"
  >
    <div
      class="header__container-logo"
    >
      <figure
        class="header__container-logo--logo"
      >
        <a
          href="/"
        >
          <img
            alt="logo"
            src="logo.png"
          />
        </a>
      </figure>
      <span
        class="header__container-logo--name"
      >
        OpenContractMX
      </span>
    </div>
    <nav
      class="navBar"
    >
      <i
        class="navBar__icon-menu"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            x1="3"
            x2="21"
            y1="12"
            y2="12"
          />
          <line
            x1="3"
            x2="21"
            y1="6"
            y2="6"
          />
          <line
            x1="3"
            x2="21"
            y1="18"
            y2="18"
          />
        </svg>
      </i>
      <ul
        class="navBar__list "
      >
        <li
          class="nav__list--home"
        >
          <a
            class="list__home--name"
            href="/"
          >
            <i
              class="list__home--icon-home fas fa-home"
            />
            Inicio
          </a>
        </li>
        <li
          class="nav__list--statistic"
        >
          <a
            class="list__statistic--name"
            href="/statistic"
          >
            <i
              class="list__statistic--icon-chart fas fa-chart-line"
            />
            EstadÃ­stica
          </a>
        </li>
        <li
          class="nav__list--about-us"
        >
          <a
            class="list__about-us--name"
            href="/about-us"
          >
            <i
              class="list__about-us--icon-team fas fa-people-carry"
            />
            Sobre Nosotros
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <footer
    class="footer"
  >
    <p
      class="footer--Copyright"
    >
      CopyrightÂ© Made with ðŸ§¡ from LATAM to the ðŸŒŽ
    </p>
  </footer>
</DocumentFragment>
`;
